exonum/src/storage/keys.rs

```diff
@@ -24,6 +26,13 @@ use crypto::{Hash, PublicKey, HASH_SIZE, PUBLIC_KEY_LENGTH};
 /// will not sort signed integer types in the natural order; therefore, they are
 /// mapped to the corresponding unsigned type by adding a constant to the source value.
 ///
+/// The `SystemTime` (de)serialization is only possible for the values greater than
```

Comment:
IMO, this paragraph better belongs to the `impl StorageKey for SystemTime` block.